docs/architecture-decisions/0011-indexing-commands-and-flows.md

```diff
@@ -0,0 +1,66 @@
+# 11. Indexing Commands and Flows
+
+Date: 2018-11-08
+
+## Status
+
+Accepted
+
+## Context
+
+Our AWS instance of Elasticsearch will be awkward to maintain unless we build the right tools into the CLI. Some of these commands will not be used often, but are anticipatory of being able to intervene and correct issues in production. As such, these are my proposed CLI commands to make available to developers / maintainers of the Discovery Index. Future needs may dictate additional commands or adjustments to these.
+
+## Decision
+
+COMMANDS:
+     ingest   Parse and ingest the input file
+							[note: was parse. Ingest is more reflective of what this does as parsing is just one aspect]
+
+		 ping     request and display general info about the Elasticsearch server
+
+     help, h  Shows a list of commands or help for one command
+
+   Index actions:
+
+     create   Create an Elasticsearch index
+
+     delete   Delete an Elasticsearch index
+
+     promote  add alias prod to provided index
```

Comment:
The wording here is awkward. I'd suggest something like "Add the given index to production alias"